app/models/government.rb

```diff
@@ -1,6 +1,9 @@
 class Government < ApplicationRecord
+  include DateValidation
   include PublishesToPublishingApi
 
+  date_attributes(:start_date, :end_date)
```

Comment:
should we validate start date is before end date? Or i guess maybe not as end date can be nil?

Reply:
It's already covered on line 12, there's `validate :not_overlapping?, if: -> { start_date.present? }` which checks quite a few things so I decided not to mess about with it.